src/router/index.js

```diff
@@ -1,14 +1,14 @@
 import Vue from 'vue'
 import VueRouter from 'vue-router'
-import Home from '../views/Home.vue'
+import HomePage from '../views/HomePage.vue'
```

Comment:
Please change the import to use the source root shortcut @
`import HomePage from '@/views/HomePage.vue'`